documents/card.md

```diff
@@ -371,7 +372,7 @@ Invoice invoice = instance.invoices.notifyBy(invoiceId,medium);
 ```java
 String invoiceId = "inv_FHrXGIpd3N17DX";
 
-instance.invoices.cancel(invoiceId)
+Invoice invoice = instance.invoices.cancel(invoiceId)
```

Comment:
this cancel is also delete or why it is changed?

Reply:
this is not delete. i forget to add variable there